### ISSUE TITLE
Add msprime dependency to packaging

### ISSUE THREAD

**Issue by hyanwong** [tskit-dev/tsinfer#736]
https://github.com/tskit-dev/tsinfer/pull/731 will add msprime as a dependency so that recombination maps can be read. We need to make sure it's in the packaging requirements for conda.

**Comment by benjeffery** [comment]
Do you know the version requirement here? I'm assuming 1.0.0?

**Comment by benjeffery** [comment]
Also if #731 is adding msprime then it the added dependancy should be part of that PR.

**Comment by hyanwong** [comment]
Yes, we need msprime.RateMap, which as ISTR was introduced in 1.0.0 Iby me!)

**Comment by hyanwong** [comment]
> Also if #731 is adding msprime then the added dependancy should be part of that PR.

Happy to do that. I'm about to update that PR. Where do I add the dev?

**Comment by benjeffery** [comment]
Wait till #737 lands, then add it in `setup.cfg`.

**Comment by hyanwong** [comment]
Done in https://github.com/tskit-dev/tsinfer/pull/731/commits/e4bc1dc12f5fe794f833eaf59c8e2d1daec1704e
